src/pages/home/report/ReportActionItem.tsx

```diff
@@ -399,6 +400,20 @@ function ReportActionItem({
             return [];
         }
 
+        if (ReportActionsUtils.isActionableAddPaymentCard(action) && shouldRenderAddPaymentCard()) {
```

Comment:
You need to add this condition above the first one, otherwise the button won't show.

Reply:
So that was the trick 😮